### PR TITLE
Update translations via script, new German translations

### PR DESCRIPTION
**Pull request by def-**
## Checklist

- [ ] Tested the change ingame
- [ ] Provided screenshots if it is a visual change
- [ ] Tested in combination with possibly related configuration options
- [ ] Written a unit test (especially base/) or added coverage to integration test
- [ ] Considered possible null pointers and out of bounds array indexing
- [ ] Changed no physics that affect existing maps
- [ ] Tested the change with [ASan+UBSan or valgrind's memcheck](https://github.com/ddnet/ddnet/#using-addresssanitizer--undefinedbehavioursanitizer-or-valgrinds-memcheck) (optional)

### CODE REVIEW

File: data/languages/german.txt [hunk 1]
```diff
@@ -1573,3 +1567,57 @@ Loading sound files
...
+== Den Anfang eines Schnitts abschneiden
+
+Slice the end of a cut
+== Das Ende eines Schnitts abschneiden
+
+Export cut as a seperate demo
+== Schnitt als separate Demo exportieren
+
+Toggle keyboard shortcuts
+== Keyboard-Shortcuts wechseln
```
Comment (heinrich5991) [inline data/languages/german.txt:1596]: Hm, this turns demo keyboard shortcuts on and off in the demo player. "wechseln" sounds like the wrong word.

File: data/languages/german.txt [hunk 2]
```diff
@@ -1573,3 +1567,57 @@ Loading sound files
...
+== Demo verlangsamen
+
+Speed up the demo
+== Demo beschleunigen
+
+Slice the beginning of a cut
+== Den Anfang eines Schnitts abschneiden
+
+Slice the end of a cut
+== Das Ende eines Schnitts abschneiden
```
Comment (heinrich5991) [inline data/languages/german.txt:1590]: Is even the english version correct here? It marks the beginning of the cut. But it doesn't really slice it, it's just a marking?

File: data/languages/german.txt (hunk 1 above)
Comment (def-) [reply data/languages/german.txt]: Done

**Comment by heinrich5991** [comment]
Thanks for also fixing the originals. :)

bors r+

**Pull request merged**